content/zh/includes/default-storage-class-prereqs.md

```diff
@@ -1,7 +1,10 @@
-您需要有一个带有默认[StorageClass](/docs/concepts/storage/storage-classes/)的动态持续卷供应程序，或者自己[静态的提供持久卷](/docs/user-guide/persistent-volumes/#provisioning)来满足这里使用的[持久卷请求](/docs/user-guide/persistent-volumes/#persistentvolumeclaims)。
+你需要有一个带有默认 [StorageClass](/zh/docs/concepts/storage/storage-classes/)的
+[动态 PersistentVolume 供应程序](/zh/docs/concepts/storage/dynamic-provisioning/)，
+或者自己[静态的提供 PersistentVolume](/docs/user-guide/persistent-volumes/#provisioning)
+来满足这里使用的 [PersistentVolumeClaim](/docs/user-guide/persistent-volumes/#persistentvolumeclaims)。
```

Comment:
Links are 404
https://kubernetes.io/zh/docs/user-guide/persistent-volumes/#provisioning 
https://kubernetes.io/zh/docs/user-guide/persistent-volumes/#persistentvolumeclaims
So I didn't update the Chinese link in this place.

I found the `_redirects` file:
```
/docs/user-guide/persistent-volumes/     /docs/concepts/storage/persistent-volumes/ 301
```

But not sure why there is no jump in Chinese.

Links are reachable.
https://kubernetes.io/zh/docs/concepts/storage/persistent-volumes/#provisioning
https://kubernetes.io/zh/docs/concepts/storage/persistent-volumes/#persistentvolumeclaims

Reply:
Should use the reachable targets rather than the English version.
In the English upstream, we are supposed to avoid using targets that are redirected.
Every such links need a new entry in the `_redirects` file, and that is insane.
The **sole** purpose of the `_redirects` file is for users who might have a bookmark pointing to the old URL.

To find out what are the BAD links in the English upstream, run this:

```
./scripts/linkchecker.py
```

The output contains lines like this:

```
File: /Users/tengqm/k8s/src/k8s.io/website/content/en/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md
    /docs/reference/setup-tools/kubeadm/kubeadm: Link using redirect records, should use /docs/reference/setup-tools/kubeadm/ instead
```

Reply:
In this case, I think it is best to change the link address of the English upstream file to non-redirected, and then synchronize the Chinese version.
I will submit a PR in  English upstream.